app/controllers/errors_controller.rb

```diff
@@ -7,7 +7,7 @@ def not_found
     Bugsnag.notify("404") do |event|
       event.severity = "info"
 
-      event.add_metadata(:request, request.env)
+      event.add_metadata(:request, :request, request.env)
```

Comment:
Maybe we could call this one `env` to more closely match the rails data

Reply:
Hang on, wasn't this correct? Isn't `request.env` a hash?